general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -402,7 +402,7 @@ def _handle_status_codes(self, response: requests.Response) -> None:
             else:
                 message = str(response.text)
             raise gss.SuperstaQException(
-                f"Non-retriable error making request to SuperstaQ API, {message}",
+                f"Non-retriable error making request to SuperstaQ API, {message}. If you would like to contact a member of our team, email us at superstaq@infleqtion.com or join our Slack workspace with this URL: https://tinyurl.com/y7t69z9f.",
```

Comment:
FYI @dowusu-antwi @richrines1 @bharat-thotakura:

We were planning on doing this only for `Internal Server Error`s but decided to do this for all errors.

For example with this change we can get something like

`Non-retriable error making request to SuperstaQ API, the circuit you submitted contains mid-circuit measurements which is not supported on this device. If you would like to contact a member of our team, email us at superstaq@infleqtion.com or join our Slack workspace with this URL: https://tinyurl.com/y7t69z9f.`

Reply:
can we use the actual slack url instead of the tinyurl? imo it's a little sketchy

Reply:
> can we use the actual slack url instead of the tinyurl?

The slack url is long-ish. @stephanielee9 Does slack allow us to have shortened slack links?

Reply:
There doesn't seem to be a Slack-generated shortened link from what I can tell

Reply:
ah yeah just saw your comment [here](https://github.com/SupertechLabs/client-superstaq/pull/498#discussion_r1224613262)

fwiw it looks like qiskit just has their own url which redirects to their slack invite: https://ibm.co/joinqiskitslack
which i guess isn't that different from using a tinyurl (though imo still seems a bit more professional/less sketchy) so maybe it's ok

though i'm also ok with the long-ish url, at least it's obvious where it's going :)